bauble/tests/lang.rs

```diff
@@ -0,0 +1,135 @@
+use bauble::{Bauble, Ref, path::TypePath};
+
+#[derive(Bauble, PartialEq, Debug)]
+struct Test {
+    x: i32,
+    y: u32,
+}
+
+#[test]
+pub fn ref_implicit_type() {
+    bauble::bauble_test!(
+        TEST0
+        [Test]
+        ["test = lang::Test{ x: -5, y: 5 }\n\
+        r = $test"]
```

Comment:
I feel like this would be more readable as:
```suggestion
        [
            "test = lang::Test{ x: -5, y: 5 }\n\
             r = $test"
        ]
```
but not sure if rustfmt would change it